cpp/environment/CMakeLists.txt

```
@@ -0,0 +1,24 @@
+add_library(
+    _environment
+    OBJECT
+    AngularSeparation.h
+    AngularSeparation.cc
+    BondOrder.h
+    BondOrder.cc
+    LocalBondProjection.h
+    LocalBondProjection.cc
+    LocalDescriptors.h
+    LocalDescriptors.cc
+    MatchEnv.h
+    MatchEnv.cc
```

Missing `Registration.h`? Or is that not necessary?
```suggestion
    MatchEnv.cc
    Registration.h
```

---

Good catch. It's not strictly necessary since it's header only and the whole cpp/environment is included, but better to be explicit here.